pages/docs/tutorials/streetlights.md

```diff
@@ -211,11 +222,11 @@ EOT`}
 
 ### 6. Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message.
 
-# Conclusions
+# Conclusion
 
-You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you get an idea of the potential.
+You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lot of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
```

Comment:
```suggestion
You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
```

We need to put this back to how it was. "Lot" is a spelling mistake here because you do not need a singular word here. 😄